Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionPermission.h

```diff
@@ -75,6 +75,10 @@ WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionNativeMessagi
 WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA))
 WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionScripting;
 
+/*! @abstract The `sidePanel` permission requests access to the `chrome.sidePanel` APIs. */
+WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
+WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionSidePanel;
```

Comment:
Note: I wanted to condition this behind `ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)`, but the build system wouldn't allow me to include `wtf/Platform.h` in this file. Does anyone know if there's a workaround for this?

Reply:
My understanding is that we don't want to have `#if` statements in API header files - since third party apps will import those and they don't know what they mean. I'm not sure what this means for the best way to handle this though - maybe to always include them? @xeenon might have a better idea on the best way ti handle it

Reply:
We should put it in _WKWebExtensionPermissionsPrivate.h until it is ready to ship and has gone through API review.